### PR TITLE
create new events for `CreateUserInfo` and `CreateTokenPayloadInterface`

### LINKED ISSUE #4459: Create new events for `CreateUserInfo` and `CreateTokenPlayloadInterface`
Following this PR: https://github.com/amplication/amplication/pull/4379, where we change the `UserInfo.ts` and `ITokenServece.ts` to template files instead of static files - create new events for `CreateUserInfo` and `CreateTokenPlayloadInterface`

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by abrl91**
Close #4459

## PR Details

create new events for `CreateUserInfo` and `CreateTokenPayloadInterface`

## PR Checklist
- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

**Comment by abrl91** [comment]
@MoFoGo  Can you please open two issues - one for creating a doc for `CreateUserInfo` event and another for `CreateTokenPayloadInterface` event

File: packages/amplication-code-gen-types/src/plugin-events-params.ts [hunk 1]
```diff
@@ -57,6 +57,17 @@ export interface CreateEntityControllerSpecParams extends EventParams {
...
 }
+
+export interface CreateUserInfoParams extends EventParams {
+  template: namedTypes.File;
+  templateMapping: { [key: string]: any };
+  filePath: string;
+}
+export interface CreateTokenPayloadInterfaceParams extends EventParams {
+  template: namedTypes.File;
+  templateMapping: { [key: string]: any };
```
Comment (barshimi) [inline packages/amplication-code-gen-types/src/plugin-events-params.ts:68]: is it safe to make it so open ? with no defaults keys ?

Comment (yuval-hazaz) [reply packages/amplication-code-gen-types/src/plugin-events-params.ts:68]: yes, this is also the standard for all other events. 
The idea is that the user can add params to the template and the mapping object

**Review by yuval-hazaz — APPROVED** [review]

**Comment by abrl91** [comment]
I created these two issues:
https://github.com/amplication/amplication/issues/4491
https://github.com/amplication/amplication/issues/4492

**Pull request merged**